addon/services/keyboard.js

```diff
@@ -70,10 +70,10 @@ export default class KeyboardService extends Service {
   @action
   _respond(event) {
     if (this._disableOnInput && event.target) {
-      const tag = event.target.tagName;
+      const target = event.composedPath()[0] ?? event.target;
```

Comment:
Note that `composedPath` is supported in all browsers except IE. But I believe this addon does not support IE anyway, right? Otherwise we could add a better check and fall back to `event.target` if not supported...